spec/features/visitors/bad_password_spec.rb

```diff
@@ -15,14 +17,33 @@
       expect(page).to have_content(error_message)
       expect(page).to have_current_path(new_user_session_path)
     end
+    locked_at = Time.zone.at(page.get_rack_session['max_bad_passwords_at'])
```

Comment:
Nice that you found a way to get at the session from the feature specs :+1:

Reply:
Yea it was a little annoying because calling that changes page. So I had to call a visit afterwards. https://github.com/railsware/rack_session_access/issues/10 Very weird